### PR TITLE
[#2917] fix: Use * instead of {0,} in regular expressions

### PR DESCRIPTION
**Pull request by zhoukangcn**
### What changes were proposed in this pull request?

Use * instead of {0,} in regular expressions

### Why are the changes needed?

Fix: #2917

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

UT

### CODE REVIEW

File: catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static Map<String, String> extractPropertiesFromSql(String createTableSql
     String[] lines = createTableSql.split("\n");
 
     boolean isProperties = false;
-    final String sProperties = "\"(.*)\"\\s{0,}=\\s{0,}\"(.*)\",?";
+    final String sProperties = "\"(.*)\"\\s*=\\s*\"(.*)\",?";
```
Comment (yuqi1129) [inline catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java:35]: Does it looks more elegant to use `*` compared to `{0,}`, I can't tell the difference between them.

Comment (zhoukangcn) [reply catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java:35]: According to Justin's suggestion, `*` is more concis.

Comment (yuqi1129) [reply catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java:35]: OK, let's just follow his suggestion.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**